### ISSUE TITLE
Ouput Binding support for class-based entities

### ISSUE THREAD

**Issue by keesschollaart81** [Azure/azure-functions-durable-extension#860]
### Description
It would be great to be able to use output bindings for Actor methods. 

## Example
```cs
public class MyEntity
{ 
    [FunctionName(nameof(MyEntity))]
    public static Task Run([EntityTrigger] IDurableEntityContext ctx)
        => ctx.DispatchAsync<MyEntity>();

    public async Task MessageReceived(
        [Queue("myQueue")]CloudQueue myQueue)
    {
        // push a message to myQueue
    }
}
```

Alternatively something like this would work for me:

```cs
public class MyEntity
{ 
    private CloudQueue myQueue;

    [FunctionName(nameof(MyEntity))]
    public static Task Run(
        [EntityTrigger] IDurableEntityContext ctx,
        [Queue("myQueue")]CloudQueue myQueue)
    {
        this.myQueue = myQueue;
        ctx.DispatchAsync<MyEntity>();
    }

    public async Task MessageReceived()
    {
        // push a message to myQueue
    }
}
```

Or even via the constructor as described in #855

**Comment by cgillum** [comment]
@ConnorMcMahon this might be a good one for you to look into as well. I suspect this will be _less_ straight forward than supporting DI, but maybe we can do some creative things to make this work without requiring support from the Functions runtime, like passing parameters to `DispatchAsync`? We should probably get @fabiocav's opinion on this as well. 

FYI @jeffhollan

**Comment by jeffhollan** [comment]
Thanks - setup meeting tomorrow for us to chat

**Comment by cgillum** [comment]
This has been merged and will be part of the v2.0.0-beta2 release.
